From b92c477f27832710cdfe20b24088b37f6f44c392 Mon Sep 17 00:00:00 2001
From: Ralf Habacker <ralf.habacker@freenet.de>
Date: Tue, 29 Sep 2020 20:09:51 +0200
Subject: [PATCH 02/17] qtpaths: Add missing location types

Task-number: QTBUG-87058
Change-Id: I58926eec16f51640f0a1cbc390167346b42290a8
(cherry picked from commit bc75dce3583962d46983aa22c971ce85b9bd4af5)
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@qt.io>
---
 src/qtpaths/qtpaths.cpp | 20 +++++++++++++-------
 1 file changed, 13 insertions(+), 7 deletions(-)

diff --git a/src/qtpaths/qtpaths.cpp b/src/qtpaths/qtpaths.cpp
index 340a2ac4..eaff69e8 100644
--- a/src/qtpaths/qtpaths.cpp
+++ b/src/qtpaths/qtpaths.cpp
@@ -89,20 +89,26 @@ static const struct StringEnum {
     const char *stringvalue;
     QStandardPaths::StandardLocation enumvalue;
 } lookupTableData[] = {
+    { "AppConfigLocation", QStandardPaths::AppConfigLocation },
+    { "AppDataLocation", QStandardPaths::AppDataLocation },
+    { "AppLocalDataLocation", QStandardPaths::AppLocalDataLocation },
     { "ApplicationsLocation", QStandardPaths::ApplicationsLocation },
+    { "CacheLocation", QStandardPaths::CacheLocation },
+    { "ConfigLocation", QStandardPaths::ConfigLocation },
+    { "DataLocation", QStandardPaths::DataLocation },
     { "DesktopLocation", QStandardPaths::DesktopLocation },
     { "DocumentsLocation", QStandardPaths::DocumentsLocation },
+    { "DownloadLocation", QStandardPaths::DownloadLocation },
     { "FontsLocation", QStandardPaths::FontsLocation },
-    { "MusicLocation", QStandardPaths::MusicLocation },
-    { "MoviesLocation", QStandardPaths::MoviesLocation },
-    { "PicturesLocation", QStandardPaths::PicturesLocation },
-    { "HomeLocation", QStandardPaths::HomeLocation },
     { "GenericCacheLocation", QStandardPaths::GenericCacheLocation },
+    { "GenericConfigLocation", QStandardPaths::GenericConfigLocation },
     { "GenericDataLocation", QStandardPaths::GenericDataLocation },
+    { "HomeLocation", QStandardPaths::HomeLocation },
+    { "MoviesLocation", QStandardPaths::MoviesLocation },
+    { "MusicLocation", QStandardPaths::MusicLocation },
+    { "PicturesLocation", QStandardPaths::PicturesLocation },
     { "RuntimeLocation", QStandardPaths::RuntimeLocation },
-    { "ConfigLocation", QStandardPaths::ConfigLocation },
-    { "GenericConfigLocation", QStandardPaths::GenericConfigLocation },
-    { "DownloadLocation", QStandardPaths::DownloadLocation }
+    { "TempLocation", QStandardPaths::TempLocation }
 };
 
 /**
-- 
2.31.1

